blog.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <title>Adopt a dog home page</title>
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+  </head>
+  <body>
+    <header>
+      <div>
+        <div>
+          <img src="images/adoptadog-logo.svg" alt="a white dog logo image for header">
+          <h1>Seattle’s premiere dog adoption agency</h1>
+        </div>
+        <div>
+          <div class="cart-icon"><img src="images/cart-icon.svg" alt="a cart icon"></div>
```

Comment:
more unneeded div's. I won't comment on all of them in your site, but for small items like an image or something you don't need to wrap in a div. A div is only really meant for groups of elements (generally speaking, there are always exceptions, but less divs is better)